test/prepare.test.js

```diff
@@ -189,32 +189,6 @@ test('Preserve indentation and newline', async (t) => {
   t.deepEqual(t.context.log.args[0], ['Write version %s to package.json in %s', '1.0.0', cwd]);
 });
 
-test('Use default indentation and newline if it cannot be detected', async (t) => {
-  const cwd = tempy.directory();
-  const npmrc = tempy.file({name: '.npmrc'});
-  const packagePath = path.resolve(cwd, 'package.json');
-  await outputFile(packagePath, `{"name": "package-name","version": "0.0.0-dev"}`);
-
-  await prepare(
-    npmrc,
-    {},
-    {
-      cwd,
-      env: {},
-      stdout: t.context.stdout,
-      stderr: t.context.stderr,
-      nextRelease: {version: '1.0.0'},
-      logger: t.context.logger,
-    }
-  );
-
-  // Verify package.json has been updated
-  t.is(await readFile(packagePath, 'utf-8'), `{\n  "name": "package-name",\n  "version": "1.0.0"\n}\n`);
-
-  // Verify the logger has been called with the version updated
-  t.deepEqual(t.context.log.args[0], ['Write version %s to package.json in %s', '1.0.0', cwd]);
-});
```

Comment:
Why did you remove this test?

Reply:
Because it was testing npm internal stuff. Npm v7 does not format `package.json` files anymore, so this test failed because no newlines were present in the expected `package.json`.
I could have "fixed" the test and adjusted the expected output, but the test before this one does exactly the same thing then and the testname would have not really described anymore what it is testing. So I decided to remove it.

Reply:
got it, thanks 👍🏼